package.json

```diff
@@ -9,7 +9,7 @@
     "serve": "node server.js",
     "start": "npm run build && npm run serve",
     "test": "jest --coverage --coverageReporters=text-lcov | coveralls",
-    "test:dev": "jest --coverage"
+    "test:dev": "jest"
```

Comment:
Please add the `--coverage` flag back. We want to see a report when we run it locally?.

Reply:
ok